### ISSUE TITLE
Bug: Impossible to adjust height/width of a portal if option 'Click anywhere on Header to Expand/Minimize' is set

### ISSUE THREAD

**Issue by mschomm** [Ross-Patterson/Portals-Desktop-Organization#211]
if the Portal Behaviour option 'Click anywhere on Header to Expand/Minimize' is set, portals will show/hide if you move the mouse over the header area. Expanding portals works fine in this mode but unfortunately the code for minimizing them again is executed without giving the user a chance to resize the portal - you simply can't grab the borders.
The only currently available way to actually resize a portal configured this way is to disable this option, resize the portal, and enable it again.

**Comment by Ross-Patterson** [comment]
I think you mean the Auto-Expand setting, not  'Click anywhere on Header to Expand/Minimize' .

But yes, when you hover over the edge of a portal and the cursor changes to the resize cursor, this triggers a MouseLeave event in Windows and this is the event that triggers the portal to minimize.

This is an API limitation that I cannot overcome unfortunately.

**Comment by Ross-Patterson** [comment]
one way of getting around this would maybe be that once you are hovered over and it expands, I could implement that if you hold Alt for example, the mouse can leave without the Portal minimizing. Then when Alt is released if the mouse is outside it minimizes.

This would allow you to hold Alt and do your resizing, then release alt when done.

**Comment by mschomm** [comment]
> I think you mean the Auto-Expand setting, not 'Click anywhere on Header to Expand/Minimize' .
> 
> But yes, when you hover over the edge of a portal and the cursor changes to the resize cursor, this triggers a MouseLeave event in Windows and this is the event that triggers the portal to minimize.
> 
> This is an API limitation that I cannot overcome unfortunately.

Right, I meant 'Auto-Expand'. BTW: I don't know how Stardock implemented it but in Fences resizing of an auto-expand fence is possible (see video).
https://github.com/Ross-Patterson/Portals-Desktop-Organization/assets/28079144/1bd9115d-34f0-42d8-8662-b09a6add41d7
